lib/compiler/passes/generate-bytecode.js

```diff
@@ -225,7 +225,10 @@ function generateBytecode(ast) {
     return Array.prototype.concat.apply([], arguments);
   }
 
-  function buildCondition(condCode, thenCode, elseCode) {
+  function buildCondition(match, condCode, thenCode, elseCode) {
+    if (match > 0) { return thenCode; }
+    if (match < 0) { return elseCode; }
```

Comment:
it's not clear what this does, why the API is replaced, what these arguments mean, why "match" being above or below zero does these things, et cetera.

Reply:
"it's an internal api" isn't a good reason for unexplained changes with obcure names that take arithmetic approaches to non-arithmetic problems

this is ioccc code